hr_holidays_validator_list/models/hr_employee.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class HrEmployeeBase(models.AbstractModel):
+    _inherit = "hr.employee.base"
+
+    leave_manager_ids = fields.Many2many(
+        "res.users",
+        string="Time Off Managers",
+        store=True,
+        readonly=False,
+        help="""Select the users responsible for approving 'Time Off' of this employee.
+        If empty, the approval is done by an Administrator
+        or Approver (determined in settings/users).""",
+    )
+
+    @api.depends("parent_id", "leave_manager_ids")
+    def _compute_leave_manager(self):
+        employees = self.filtered(lambda e: e.leave_manager_ids)
+        for employee in employees:
+            employee.leave_manager_id = employee.leave_manager_ids[0]
+        remaining_employees = self - employees
+        return super(HrEmployeeBase, remaining_employees)._compute_leave_manager()
+
+    def _add_leave_manager_ids_in_group(self):
+        approver_group = self.env.ref(
+            "hr_holidays.group_hr_holidays_responsible", raise_if_not_found=False
+        )
+        if not approver_group:
+            return
+        managers = self.mapped("leave_manager_ids")
+        if managers:
+            approver_group.users |= managers
```

Comment:
```suggestion
            approver_group.sudo().users |= managers
```
@Kimkhoi3010  You will need to use sudo here otherwise you get an access error on res.groups if you're not an admin user. Could you fix the pr ? Thanks

Reply:
Hi @victorvermot, I have updated the PR.
Thanks for your time.